coralproject/talk
# [CORL-235] Allow a user to download their comment history

## What does this PR do?

Allows a user to request to download their comments. This will trigger an email that they can then use to hit a landing page to download a CSV of their comment history.

## How do I test this PR?

- Have email development setup via Inbucket or similar
- Click on My Profile > Settings in the comment stream tabs
- Select "Request comment history"
- Check your email service for the email sent out
- Click the link in the email
- You should appear on a landing page to download your comments
- Select "Download My Comment History"
- An archive of your comments should download
- Unzip the archive and verify the comment data for accuracy
